### PR TITLE
Update to the latest version of DocSearch

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

**Comment by nex3** [comment]
Preview is up at https://staging-sass-lang.herokuapp.com/

**Review by Awjin — APPROVED** [review]

File: source/assets/css/regions/_banner.scss [hunk 1]
```diff
@@ -40,4 +40,11 @@
       border: 0;
     }
   }
-}
\ No newline at end of file
+}
+
+#docsearch {
```
Comment (jina) [inline source/assets/css/regions/_banner.scss:45]: This is a nitpick, but I usually like to separate out styles that are specific to third-party stuff into it's own partial. Especially when it relies on IDs to target.

**Review by jina — CHANGES_REQUESTED** [review]
I just had one comment but otherwise everything else looks good. I'll put "request changes" for now to see what you think of my comment.

File: source/assets/css/regions/_banner.scss (hunk 1 above)
Comment (nex3) [reply source/assets/css/regions/_banner.scss:45]: Sure! Where would you like it to go?

Comment (jina) [reply source/assets/css/regions/_banner.scss:45]: how about `source/assets/css/vendor/_docsearch.scss`?

Comment (nex3) [reply source/assets/css/regions/_banner.scss]: Made my best guess. Feel free to move it around after this lands.

Comment (jina) [reply source/assets/css/regions/_banner.scss]: did you not like my suggestion? (no worries. can move it later) :)

**Comment by nex3** [comment]
I moved it in
https://github.com/sass/sass-site/pull/593/commits/6d4c2d2ca45a787c78b34d9af3783e0bdbb93175
:)

**Comment by jina** [comment]
I was meaning my suggestion as to where to put it. https://github.com/sass/sass-site/pull/593#discussion_r743336791

You said you made your best guess but I had suggested a location. But no worries. I plan to refactor the styles eventually anyway so I can do some clean up later. :)

**Comment by nex3** [comment]
Oh, I just missed your suggestion! Sorry about that!

**Pull request merged**